einsumt.py

```diff
@@ -123,8 +123,9 @@ def einsumt(*operands, **kwargs):
         for j in range(len(ops)):
             oj = ops[j]
             if cpos[j] >= 0:
-                jslice = cpos_slice[j] + (islice,)
-                oj = oj[jslice]
+                if oj.shape[cpos[j]] > 1:
+                    jslice = cpos_slice[j] + (islice,)
+                    oj = oj[jslice]
```

Comment:
Apply the slice `jslice` on the j-th operand only if the dimension of the chunk-component is greater than one. Otherwise, take the full operand.